lexibank_yanglalo.py

```diff
@@ -1,82 +1,72 @@
-import csv
-
-from clldutils.path import Path
-from clldutils.text import split_text
-from pylexibank.dataset import NonSplittingDataset, Language, Concept
+from pathlib import Path
+from pylexibank.dataset import Dataset as BaseDataset
+from pylexibank import Language, Concept
+from pylexibank import FirstFormOnlySpec
 from pylexibank.util import pb
+
+from clldutils.misc import slug
 import attr
 
 @attr.s
-class HLanguage(Language):
+class CustomLanguage(Language):
     Location = attr.ib(default=None)
     Source_ID = attr.ib(default=None)
+    Family = attr.ib(default='Sino-Tibetan')
+    SubGroup = attr.ib(default='Lalo')
 
 @attr.s
-class HConcept(Concept):
+class CustomConcept(Concept):
     Chinese_Gloss = attr.ib(default=None)
 
 
 
-class Dataset(NonSplittingDataset):
+class Dataset(BaseDataset):
     id = "yanglalo"
     dir = Path(__file__).parent
-    concept_class = HConcept
-    language_class = HLanguage
-
-    def cmd_download(self, **kw):
-        # nothing to do, as the raw data is in the repository
-        pass
+    concept_class = CustomConcept
+    language_class = CustomLanguage
+    form_spec = FirstFormOnlySpec(missing_data=('烂饭', '-'))
```

Comment:
This is the important point. If you want to split the first entry off, use the FirstFormOnlySpec.